tech.keyforge.master.metainfo.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>tech.keyforge.master</id>
```

Comment:
"Master" doesn't seem to be the app name according to the `<name>` tag.

Reply:
I'll rephrase: the app-id doesn't feel right for the above reason. The last component should represent the application name, which "Master" is not.

Reply:
Ok, I will update the app ID

Reply:
I have updated the app ID.